samples/BindingDemo/BindingDemo.csproj

```diff
@@ -15,4 +15,7 @@
   <Import Condition="'$(TargetFramework)'=='net461'" Project="..\..\build\NetFX.props" />
   <Import Project="..\..\build\ReferenceCoreLibraries.props" />
   <Import Project="..\..\build\BuildTargets.targets" />
+  <ItemGroup>
+    <PackageReference Update="ReactiveUI" Version="11.4.17" />
```

Comment:
Hello!

Shouldn't we also update the following?

https://github.com/AvaloniaUI/Avalonia/blob/70a0f00422b6638492e1bbf3c7de847bc8ac01c9/build/ReactiveUI.props#L3